game/scripts/npc/heroes/huskar.txt

```diff
@@ -0,0 +1,10 @@
+"DOTAHeroes"
+{
+  //=================================================================================================================
+  // HERO: Huskar
+  //=================================================================================================================
+  "npc_dota_hero_huskar"
+  {
+    "Ability16"                                   "special_bonus_attack_range_200" // replaces life break talent         
```

Comment:
life break talent will either be garbage or OP in oaa with the previous scaling of huskar ulti. It's better to just revert back to his old talent.

Reply:
I approve the removing of life break talent but +200 attack range still seems bad.

Reply:
what about this? @DarkoniusXNG Bring it back to the good ol days when Huskar had magic resistance on his berserker's blood. Can't actually incorporate that into the ability but what do you say for this talent?